salt/modules/hashutil.py

```diff
@@ -113,23 +113,23 @@ def base64_b64decode(instr):
     return salt.utils.hashutils.base64_b64decode(instr)
 
 
-def base64_encodestring(instr):
+def base64_encodebytes(instr):
```

Comment:
We don't want to just drop the existing name - because whatever code that's currently using it will fail without warning.

What I would do is actually just 1) create encodebytes like you have, but 2) leave the encodestring and it should basically be:

    return salt.utils.stringutils.to_str(salt.utils.hashutils.base64_encodebytes(instr))

(It might be that we need to also `salt.utils.stringutils.to_bytes(instr)` there - I'd have to look a bit closer)

Reply:
Completed